walkthroughs/week-3-server/examples/text-processor/src/main/java/com/google/sps/servlets/TextProcessorServlet.java

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC
+ // Copyright 2019 Google LLC
```

Comment:
Remove space?

Reply:
Oops! This was an example file that I did not mean to edit - fixed!